doc/source/changes/version_0_25_2.rst.inc

```diff
@@ -0,0 +1,15 @@
+Miscellaneous improvements
+--------------------------
+
+* ...
+
+Fixes
+-----
+
+* fixed extracting LArray object using `from_string` function when second axis is implicit
```

Comment:
It is much larger than just from_string, which most users do not know/care about. My test case used from_string so that it was self-contained, but the initial bug report was using a .csv file and I suspect excel would suffer from this too (and depending on the backend, this fix might not affect it because the code path is very different with the xlwings backend).